exercises/exercise-1/exercise-1.js

```diff
@@ -3,8 +3,8 @@ const personOne = {
   age: 34,
   favouriteFood: "Spinach",
 };
-
-function introduceYourself(___________________________) {
+const{name, age, favouriteFood} = personOne;
+function introduceYourself(object) {
```

Comment:
I would suggest moving this into the param
```suggestion
function introduceYourself({name, age, favouriteFood}) {
```